tachyon/zk/plonk/circuit/region.h

```diff
@@ -98,6 +98,91 @@ class Region {
       return Error::kNone;
     }
   };
+
+  explicit Region(Layouter* layouter) : layouter_(layouter) {}
+
+  // See the comment above.
+  Error EnableSelector(AnnotateCallback annotate, const Selector& selector,
+                       size_t offset) {
+    return layouter_->EnableSelector(std::move(annotate), selector, offset);
+  }
+
+  // See the comment above.
+  void NameColumn(AnnotateCallback annotate, const AnyColumn& column) {
+    layouter_->NameColumn(std::move(annotate), column);
+  }
+
+  // See the comment above.
+  Error AssignAdvice(AnnotateCallback annotate, const AdviceColumn& column,
+                     size_t offset, AssignCallback assign,
+                     AssignedCell<F>* assigned_cell) {
+    Cell cell;
+    Value<F> value = Value<F>::Unknown();
+    Error error = layouter_->AssignAdvice(
+        std::move(annotate), column, offset,
+        [&value, assign = std::move(assign)]() {
+          value = std::move(assign).Run();
+          return value.ToRationalFieldValue();
+        },
+        &cell);
+    if (error != Error::kNone) return error;
+    *assigned_cell = {cell, value};
+    return Error::kNone;
+  }
+
+  // See the comment above.
+  Error AssignAdviceFromConstant(AnnotateCallback annotate,
+                                 const AdviceColumn& column, size_t offset,
+                                 const F& constant,
+                                 AssignedCell<F>* assigned_cell) {
+    Cell cell;
+    Error error = layouter_->AssignAdviceFromConstant(
+        std::move(annotate), column, offset, constant, &cell);
+    if (error != Error::kNone) return error;
+    *assigned_cell = {cell, Value<F>::Known(constant)};
+    return Error::kNone;
+  }
+
+  // See the comment above.
+  Error AssignAdviceFromInstance(AnnotateCallback annotate,
+                                 const InstanceColumn& instance, size_t row,
+                                 const AdviceColumn& advice, size_t offset,
+                                 AssignedCell<F>* cell) {
+    return layouter_->AssignAdviceFromInstance(std::move(annotate), instance,
+                                               row, advice, offset, cell);
+  }
+
+  // See the comment above.
+  Error AssignFixed(AnnotateCallback annotate, const FixedColumn& column,
+                    size_t offset, AssignCallback assign,
+                    AssignedCell<F>* assigned_cell) {
```

Comment:
```suggestion
                    AssignedCell<F>* assigned_cell) override {
```
Plus, if we get to add override keyword, how about removing " // See the comment above." comment?